### DISCUSSION TITLE
How to use drag and drop

### DISCUSSION THREAD

**Discussion by huajuhong** [microsoft/fluentui-blazor#1938] [Q&A] [answered]
Why was the drag related event not triggered when I used the drag function in reference to the document
![image](https://github.com/microsoft/fluentui-blazor/assets/35752873/014eb47a-6f84-4b1d-a576-95f751cc007a)
Drog and Drop is invalid，`OnDragEnter/OnDragLeave/OnDropEnd` Not triggered

**Comment by vnbaaij** [comment]
Well, you can see it working in the demo site:

![image](https://github.com/microsoft/fluentui-blazor/assets/1761079/98aed1c8-eac1-4bf8-834f-72c33235ea1e)

**Comment by oneolddev** [comment, marked as answer]
@Hsiaye 

I believe you haven't set an interactive rendermode.

``` razor
@rendermode InteractiveServer
```

**Reply by huajuhong** [reply]
Think you, this is successful
